small genomes too (#373)
bcgsc/abyss: README.md

=====

ABySS is a *de novo* sequence assembler intended for short paired-end

> reads. Initially designed for large genomes, ABySS also performs a decent job assembling small (e.g. bacterial) genomes.

Reviewer: ```suggestion
reads. Initially designed for large genomes, ABySS also performs well on small (e.g. bacterial) genomes.
```
Or some thing like that? I wasn't sure about the phrase "decent job"

Author: My only concern is that people may read this and think "well, they mean it does work on smaller genomes but probably they're not the best". But we know we're doing as good as other alternatives... so how do you think we can make this impression?

Author: @lcoombe, I've seen that on some well-known repos people add a comparison too.. like a small table showing their results side-by-side of another tool well-used by the community... that'd show everything but I am not sure if we like that sort of thing in a repo readme

Reviewer 2: ABySS is a *de novo* sequence assembler intended for short paired-end reads and genomes of all sizes.

Reviewer: Thanks Shaun - I like this better!
I think updating that text is fine for now, if desired we could add more in a future PR.

Author: Thanks @sjackman, I like your idea! It's simple and right! we don't need to give a background of what abyss was designed for in here!
